data/core/units/elves/Hero.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Elvish Champion
     cost=30
     usage=fighter
-    description= _ "Elves, though not warlike by nature, are swift to learn; a relatively small amount of experience will turn an elf from a competent fighter into a master of combat. Those who are honored as heroes are strong with both sword and bow, and their skill once acquired does not fade for lack of practice."
+    description= _ "Elves that have grown more adept at swordsmanship often forego a shield in favor of a secondary shortsword, useful for both parrying enemy attacks and sneaking in extra strikes when engaging in melee combat. These diminutive blades are often known as 'Váthial', little heroes, for their third use as replacement swords for weaponless allies. More inexperienced fighters are easily disarmed by stronger opponents and quickly overwhelmed when only left with a bow to fight with. By offering their parrying swords to their more vulnerable comrades, elvish heroes grant them a second chance to fight or to retreat with their lives."
```

Comment:
```suggestion
    description= _ "Accomplished Elvish warriors are sometimes called ‘Sárkan’, roughly meaning ‘one who has sacrificed in defense’ or ‘one transfigured by experience’. Often translated to ‘Elvish Heroes’ in the human tongue, this name does indicate some level of esteem and prestige, but is not used to glorify the position that these elves hold. Elvish Heroes are those who have survived numerous wars and have outlived many of their friends who fell in prior battles. They have seen all manner of atrocity committed by all races, be it orcs and humans ravaging the natural beauty of the forests, or even their own kind, other elves who are not above slaughtering innocent people out of pride and vengeance. Many years of fighting naturally make these swordsmen highly skilled in melee combat, but the hardships and destruction wrought by war take a great toll on both mind and body. While the Elves realize the necessity of military strength and honor the ‘Sárkanet’ as heroes for their service, such a status is not one often sought after by the majority of Elvenkind."
```

Reply:
So I made this new description, but still need help with the Elvish word (it's a corruption of the Finnish word for hero, which is apparently 'sankari' according to google translate). I would like to know if the plural form is correct in the description, or if someone else has a better name that they'd like to suggest.

Reply:
How about no name, especially considering you're using Google Translate instead of being a native speaker?

Where's my facepalm emote? /s

Reply:
@GregoryLundberg Let's not throw the baby out with the bathwater. If using a made-up word is better than not, then let's keep the made-up word but run it by native speakers.

Reply:
I don't see what the problem with making names is. We do it everywhere else for naming locations, towns, units, and characters (and it's not like those magically popped out of nowhere). I don't mind if people change the word; that's precisely why I'm asking if anyone has any suggestions here.

Reply:
Well, as I said, be sure to include a PO comment so translators know your intention and that they can choose an alternate name if needed.

Reply:
1. I'm distinctly a little disappointed at the removal of "Váthial" (or did I miss it? It's hard to read these changes in the main conversation timeline).
2. I don't think you need to worry about accuracy to the Finnish. The Elvish language is not Finnish it's something else, perhaps something that bears some resemblance to Finnish, sure, but not actually Finnish. If it sounds good and doesn't contradict established Elvish phonotactics (and I don't think "Sárkan" does, though I'd spell it with a C instead of a K), then I'm fine with it.

Reply:
Having more properly read the proposed change now, I'm of the opinion that the plural is fine. I don't know if it conforms to Finnish morphology, but it doesn't have to &ndash; it's not Finnish, like I said in the previous comment.

I'm still slightly disappointed at the word I recommended being remove, but I guess I wasn't all that attached to it, and perhaps an explanation of the unit's name is indeed better than inventing a word to describe their secondary sword.

In short: I approve of this suggestion. The only change I'd recommend changing is to spell it with C instead of K, as the elves don't seem to use K (judging from the names, at least).

Reply:
If y'all want to restore that other word, go ahead.
Be sure to add PO comments about the intended meaning.

I'll be giving this PR a couple more days to settle down then collapsing it to a single commit merging, and cherry-picking to master. After last night, I'm thinking Github needs the time off, too. They still have some issues to iron out, it seems.